src/dialects/postgres/query-generator.js

```diff
@@ -401,7 +401,9 @@ export class PostgresQueryGenerator extends AbstractQueryGenerator {
     const limit = options.limit ? ` LIMIT ${this.escape(options.limit, undefined, _.pick(options, ['replacements', 'bind']))}` : '';
     let primaryKeys = '';
     let primaryKeysSelection = '';
-
+    const baseQuery = `DELETE FROM ${table}`;
+    let query = '';
```

Comment:
declare query on line 421 & 430 instead, it's not used anywhere else (we aim to declare variables as "late" as possible)